variables.tf

```diff
@@ -3,8 +3,21 @@
 ##############################################################################
 
 variable "name" {
-  description = "Name for VPC"
+  description = "Name for VPC."
```

Comment:
Perhaps update to: "The name to give the newly provisioned VPC. Only used if 'create_vpc' is true."